Dockerfile

```diff
@@ -7,6 +7,20 @@ ENV NPM_BUILD_COMMAND=compile
 ARG BUILD_VERSION
 ENV BUILD_VERSION="$BUILD_VERSION"
 
+USER root
+
+ARG UID=1000
```

Comment:
Will docker or docker compose pick up `UID` / `GID`  from the environment automatically? 

If environment variables have to be changed or updated to support this, can you update the documentation as well, explaining the process for the full fix?